consensus/blake3pow/consensus.go

```diff
@@ -238,14 +240,15 @@ func (blake3pow *Blake3pow) verifyHeader(chain consensus.ChainHeaderReader, head
 			return consensus.ErrFutureBlock
 		}
 	}
-
 	if header.Time() < parent.Time() {
 		return errOlderBlockTime
 	}
-
+	// Make sure subordinate difficulty does not exceed local difficulty
+	if nodeCtx < common.ZONE_CTX && header.Difficulty().Cmp(header.Difficulty(nodeCtx+1)) < 0 {
```

Comment:
Instead of common.ZONE_CTX, I think we should use common.HierarchyDepth-1 which would more clearly signify the bounds checking intention.

Reply:
good idea!